### ISSUE TITLE
In the "All contents" pane, a share icon is misleading

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#2804]
## Bug description & symptom

In the "All contents" pane, a share icon is used to represent received files.
This icon is used everywhere in other user interfaces to share something, on buttons and links. I think this use of the share icon is misleading.

What do you think @celiamgtt?

## Expected

An icon better representing  the idea of "received files".

## Screenshots

![image](https://user-images.githubusercontent.com/3817365/77999819-93e4e780-7333-11ea-9b78-e6d991debfb9.png)

## Impacted components

- *frontend*

**Comment by celiamgtt** [comment]
I agree, it's unclear. Do you have any idea about the icon we could use? (it has to be explicit and not being used for another action/feature)
The list of available icons is this one:
https://fontawesome.com/v4.7.0/cheatsheet/

I don't know if it's still something we are discussing about, but we talked about having a link in the sidebar 'received content' or 'shared content' instead of the current folder.

**Comment by raphj** [comment]
What about one of these? Any preference?

 - fa-download
 - fa-paperclip

**Comment by PhilippeAccorsi** [comment]
>     * fa-download
Used to download file in tracim (since begining of tracim). Here the feature is not to download files.

>     * fa-paperclip
Used for files in tracim (since begining of tracim).

When we have make this feature, we not have found good icon (discussion between algoo employes), so I think we need to open discussion whis all tracim users. Discuss whis @LucBarry to find a good way for this. Maybe request on Twitter or something like that.

When we have make this feature, we not have found good icon (discussion between algoo employes), so I think we need to open discussion whis all tracim users. Discuss whis @LucBarry to find a good way for this. Maybe request on Twitter or something like that.

**Comment by PhilippeAccorsi** [comment]
To create links for external user you need to open option in this "box" (like we have to share files).
In advanced dashboard, feature available are :
- Download activated
- Upload deactivated
